superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -559,15 +559,16 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
         onChange={setDatabaseModel}
         placeholder="Choose a database..."
       >
-        {availableDbs?.databases
-          ?.sort((a: DatabaseForm, b: DatabaseForm) =>
-            a.name.localeCompare(b.name),
-          )
-          .map((database: DatabaseForm) => (
-            <Select.Option value={database.engine} key={database.engine}>
-              {database.name}
-            </Select.Option>
-          ))}
+        {availableDbs &&
```

Comment:
since you have the conditional operator below, you don't need to check for availableDbs again here.